components/loader/src/circular-loader/circular-loader.js

```diff
@@ -67,6 +69,7 @@ CircularLoader.defaultProps = {
 }
 
 CircularLoader.propTypes = {
+    ariaLabel: PropTypes.string,
```

Comment:
@cooper-joe should the `ariaLabel` be required here?

Reply:
No, I don't think so. As I understood the requirements, an accessible label can be provided alongside the loader in a container. So in that case, `ariaLabel` isn't technically required. And for the purposes of adoption, I'd prefer to throw lint or console warnings for these violations rather than prop errors.